tests/rados/test_bluestoretool_workflows.py

```diff
@@ -490,8 +703,18 @@ def run(ceph_cluster, **kw):
         log.info(
             "\n \n ************** Execution of finally block begins here *************** \n \n"
         )
+        # fail-safe removal of all OSDs from node13
```

Comment:
please also delete the VG and the PV created for the test.

LV should be deleted when OSD removal is done with "--zap"